test/evergreen.yml

```diff
@@ -307,11 +310,22 @@ functions:
           powershell.exe  -NonInteractive '.\test\evergreen\build_windows.ps1' -configure 1 $DEFINED_EVERGREEN_CONFIG_FLAGS ${windows_configure_flags|}
         else
           echo "Using config flags $DEFINED_EVERGREEN_CONFIG_FLAGS ${posix_configure_flags|}"
-          # Fetch the gperftools library if needed. This will also get tcmalloc.
+
           if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]]; then
-            is_cmake_build=true
-            . ../automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} $is_cmake_build
```

Comment:
If you're dropping this, you should also be able to remove the `get automation-scripts` step anywhere we call `configure wiredtiger`

Reply:
Yes, thanks, removed.